tutorials/spa-create-excel-automation/spa-create-excel-automation.md

```diff
@@ -0,0 +1,199 @@
+---
+author_name: Ramakrishnan Raghuraman
+author_profile: https://github.com/r3ksk
+title: Build your first automation using Excel SDK of SAP Process Automation
+description: In this tutorial, you will be developing basic automations using Excel SDK, In the end, you will know how to read excel data for a given range using **Excel Cloud Link**, or using traditional approach. In this tutorial, we will show how you can apply filter on the excel data, copy the filtered data onto a new excel sheet and save the file with a new name.
+auto_validation: true
+time: 30
+tags: [ tutorial>beginner, software-product>sap-business-technology-platform, tutorial>free-tier]
+primary_tag: software-product>sap-process-automation
+---
+
+## Details
+
+SAP Process Automation has native integration to several Microsoft Office products including Outlook, Excel SDK. In this Tutorial, you will explore how to use Excel SDK to your automations.
+
+## Prerequisites
+
+- Complete the tutorial: [Subscribe to SAP Process Automation Using Booster in SAP BTP Free Tier](spa-subscribe-booster)
+- Complete the tutorial: [Install and Set Up the Desktop Agent](spa-setup-desktop-agent)
+- Familiarity to create an automation using SAP Process Automation
```

Comment:
How does one gain “Familiarity to create an automation using SAP Process Automation”?  Is there a tutorial or group of tutorials that can help the user get familiar?

Reply:
There is no single tutorial like this. I will have to include a blog or video from our community to explain my ask here. tutorials and missions in develoepr tutorials talk other things and it will confuse the ask.

Reply:
I will also check with Priya in this regard

Reply:
based on Priya's advice I have amended the text, let me know if this works for you.